client/src/app/core/ui-services/applause.service.ts

```diff
@@ -0,0 +1,47 @@
+import { Injectable } from '@angular/core';
+
+import { Observable } from 'rxjs';
+import { distinctUntilChanged, map } from 'rxjs/operators';
+
+import { HttpService } from '../core-services/http.service';
+import { NotifyBase, NotifyService } from '../core-services/notify.service';
+
+export interface Applause {
+    message: {
+        level: number;
+        presentUsers: number;
+    };
+    name: string;
+    sendByThisUser: boolean;
+    sender_channel_id: string;
+    sender_user_id: number;
+}
+
+@Injectable({
+    providedIn: 'root'
+})
+export class ApplauseService {
+    private applausePath = '/system/applause';
+    private applauseNotifyPath = 'applause';
+
+    public applauseObservable: Observable<Applause> = this.notifyService
+        .getMessageObservable<Applause>(this.applauseNotifyPath)
+        .pipe(
+            /**
+             * forcefully map the callback from regular BaseNotify to the applause interface
+             */
+            map(callback => (callback as unknown) as Applause),
```

Comment:
You can use the following interface:
```
interface Applause {
    level: number;
    presentUsers: number;
}
```
and than `map(notify => notify.message)`. This allowes for not having `message` in e.g. the applause display component.